ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -458,7 +453,34 @@ void NGraphEncapsulateOp::ComputeUsingParallelExecutor(OpKernelContext* ctx) {
 
   // And execute
   ngraph::Event event_execute_graph("Execute Graph", "", "");
-  ng_exec->call(get<2>(io_tensors), get<1>(io_tensors));
+  BackendManager::LockBackend(m_parallel_executor->GetOpBackendName());
+  NGRAPH_VLOG(4) << "NGraphEncapsulateOp::Compute call starting for cluster "
```

Comment:
why it this diff showing up?

Reply:
Added try cacth around ng_exec-> call(), which was missing, should have been added at the time of parallel_executor execution.